srtcore/core.cpp

```diff
@@ -9860,19 +9867,23 @@ int srt::CUDT::handleSocketPacketReception(const vector<CUnit*>& incoming, bool&
 
         if (m_pRcvBuffer)
         {
-            bufinfo << " BUFr=" << avail_bufsize
-                << " avail=" << getAvailRcvBufferSizeNoLock()
-                << " buffer=(" << m_iRcvLastSkipAck
-                << ":" << m_iRcvCurrSeqNo                   // -1 = size to last index
-                << "+" << CSeqNo::incseq(m_iRcvLastSkipAck, m_pRcvBuffer->capacity()-1)
+            // XXX Fix this when the end of contiguous region detection is added.
+            int ackidx = CSeqNo::seqoff(m_pRcvBuffer->getStartSeqNo(), m_iRcvLastAck);
```

Comment:
```suggestion
            const int ackidx = CSeqNo::seqoff(m_pRcvBuffer->getStartSeqNo(), m_iRcvLastAck);
```

Reply:
The instruction two lines below this will not compile.

Reply:
Can be converted into a single line using the `std::max(0, CSeqNo...)`.